setup.py

```diff
@@ -0,0 +1,29 @@
+from setuptools import setup
```

Comment:
Without the missing data_files (like here: https://github.com/ros2/demos/blob/16dc63380acca31eddd7cd6518a416830fa53dec/demo_nodes_py/setup.py#L18) the script will not be `ros2 run`-able.

I created #4 to track the problem.